development/kops/delete_store.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+# Copyright 2020 Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+set -eo pipefail
```

Comment:
Can you upload results from a test run with set -x on for the entire script?

Reply:
```
% ./delete_store.sh kops-state-store-ibedeevblusoqrsrijfaxryjvaycnrsi
Deleting kops store s3://kops-state-store-ibedeevblusoqrsrijfaxryjvaycnrsi
+ aws s3 rm --recursive s3://kops-state-store-ibedeevblusoqrsrijfaxryjvaycnrsi
delete: s3://kops-state-store-ibedeevblusoqrsrijfaxryjvaycnrsi/./terry.k8s.local.yaml
+ aws s3api delete-bucket --bucket kops-state-store-ibedeevblusoqrsrijfaxryjvaycnrsi
```